packages/shared/src/components/auth/AuthOptions.tsx

```diff
@@ -225,7 +229,7 @@ function AuthOptions({
         <Tab label={Display.SignBack}>
           <AuthSignBack
             onRegister={() => setActiveDisplay(Display.Default)}
-            onProviderClick={onSocialRegistration}
+            onProviderClick={onProviderClick}
```

Comment:
Not sure if this was also part of the debugging phase.

Reply:
No this needs to exist for the fix on login to work (not even sure why it was a seperate action)